lnbits/core/templates/core/_api_docs.html

```diff
@@ -232,6 +232,27 @@ <h5 class="text-caption q-mt-sm q-mb-none">Curl example</h5>
         >
       </q-card-section>
     </q-card>
+    <q-card>
+      <q-card-section>
+        <code
+          ><span class="text-pink">WS</span>
+          /api/v1/ws/&lt;invoice_key&gt;</code
+        >
+        <h5
+          class="text-caption q-mt-sm q-mb-none"
+          v-text="$t('websocket_example')"
+        ></h5>
+        <code
+          >wscat -c <span v-text="websocketUrl"></span>/<span
+            v-text="wallet.inkey"
```

Comment:
- this leaks the invoice key
- we have a toggle for keys visibility to avoid over the shoulder attack

Reply:
fixed